include/boost/histogram/python/axis.hpp

```diff
@@ -260,71 +259,67 @@ py::array_t<double> widths(const A &ax) {
     return result;
 }
 
-// These match the Python names
-
-using _regular_uoflow = bh::axis::regular<double, bh::use_default, metadata_t>;
-using _regular_uflow  = bh::axis::
-    regular<double, bh::use_default, metadata_t, bh::axis::option::underflow_t>;
-using _regular_oflow = bh::axis::
-    regular<double, bh::use_default, metadata_t, bh::axis::option::overflow_t>;
-using _regular_noflow
-    = bh::axis::regular<double, bh::use_default, metadata_t, bh::axis::option::none_t>;
-using _regular_growth = bh::axis::
-    regular<double, bh::use_default, metadata_t, bh::axis::option::growth_t>;
+// These match the Python names except for a possible underscore
```

Comment:
I was debating adding the underscore here.

By the way, the new Python layout means these are only available from `boost_histogram.core.axis`, and not from the normal `boost_histogram.axis`, so dropping the underscore entirely might be an option. Up to you.

Reply:
Does it make sense to have a `_core` module and then everything inside can be without underscore?

Reply:
We can. I wasn't sure whether to name it core or _core. I don't currently have a strong reason to go one way vs. the other. I *think* I like _core + no underscores in names inside it a bit better.

Reply:
(also a topic for another PR)

Reply:
Right.